.github/workflows/ci-cron-vulnerability-scans.yml

```diff
@@ -0,0 +1,50 @@
+# GitHub Actions CI workflow that runs vulnerability scans on the application's Docker image
+# to ensure images built are secure before they are deployed.
+
+name: CI Vulnerability Scans
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Run every day at 07:00 UTC (3am ET, 12am PT) after engineers are likely done with work
+    - cron: "0 7 * * *"
```

Comment:
What about running it closer to the start of the work day (7~9am EST?) just for all those fun last-second vulnerabilities that pop up.

Reply:
"fun" 😆 

anyways, yeah sure